x-pack/test/functional/apps/infra/node_details.ts

```diff
@@ -139,40 +139,32 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
     let synthEsClient: InfraSynthtraceEsClient;
     before(async () => {
       synthEsClient = await getInfraSynthtraceEsClient(esClient);
-      await synthEsClient.clean();
       await kibanaServer.savedObjects.cleanStandardList();
       await browser.setWindowSize(1600, 1200);
-    });
 
-    after(async () => {
-      await synthEsClient.clean();
+      return synthEsClient.clean();
     });
 
+    after(() => synthEsClient.clean());
+
     describe('#Asset Type: host', () => {
       before(async () => {
-        synthEsClient = await getInfraSynthtraceEsClient(esClient);
-        await synthEsClient.clean();
         await synthEsClient.index(
           generateHostData({
             from: DATE_WITH_HOSTS_DATA_FROM,
             to: DATE_WITH_HOSTS_DATA_TO,
             hosts: HOSTS,
           })
         );
+
         await navigateToNodeDetails('host-1', 'host', {
           name: 'host-1',
         });
 
         await pageObjects.header.waitUntilLoadingHasFinished();
-        await pageObjects.timePicker.setAbsoluteRange(
```

Comment:
What's the reason why we don't need to navigate to the date with data?

Reply:
We're setting the date picker with the same information in the test cases.